packages/server/utils/getJiraCloudIdAndKey.ts

```diff
@@ -0,0 +1,5 @@
+const getJiraCloudIdAndKey = (id: string): [cloudId: string, issueKey: string] => {
```

Comment:
i moved this one to /client/utils since we're using it in both no, we could probably just use that one?